bottom/bottom.go

```diff
@@ -78,6 +112,22 @@ func calculateValueCharacters() [255]string {
 	return valueCharacters
 }
 
+var (
+	// ErrByteOverflow is returned when the given input string overflows a byte
+	// when decoded.
+	ErrByteOverflow = errors.New("sum overflows byte")
+)
```

Comment:
This feels inconsistent compared to something like:
```suggestion
// ErrByteOverflow is returned when the given input string overflows a byte
// when decoded.
var ErrByteOverflow = errors.New("sum overflows byte")
```

Reply:
True. I only did this because there was an error here previously, but I'll fix it now.